pkg/features/kube_features.go

```diff
@@ -155,6 +155,12 @@ const (
 	// Enable pods to consume pre-allocated huge pages of varying page sizes
 	HugePages utilfeature.Feature = "HugePages"
 
+	// owner: @sjenning
+	// beta: v1.11
+	//
+	// Enable pods set sysctls on pod
```

Comment:
nit: Enable pods to set sysctls on a pod.

Reply:
Done.